README.md

```diff
@@ -7,13 +12,13 @@
 4| [The Great Alexa Hackathon](https://content.techgig.com/registrations-open-for-alexa-skills-hackathon-at-code-gladiators-2019/articleshow/68922361.cms) | July 2019 | HackerEarth | Submitted, Failed |
 5| [HackIIITD](https://hackiiitd.tech/) | 23-24 August 2019 | IIITD | 4th Place Win |
 6| [Hack Your Reality](https://hackyourreality.hackerearth.com/) | August 2019 | HackerEarth | Top 12 |
-7| [NASA space apps challenge qualifiers](https://2019.spaceappschallenge.org/locations/new-delhi/) | September 2019 | Galgotias, Greater Noida | Submitted, Failed |
+7| [NASA space apps challenge qualifiers](https://2019.spaceappschallenge.org/locations/new-delhi/) | September 2019 | Galgotias, Greater Noida | 1st Place Win |
 8| [UHack](https://infoxpression.in/events/UHACK) | 12-13 Oct 2019 | GGSIPU, Dwarka | Submitted, Failed |
 9| [HackCBS2.0](https://hackcbs.tech/) | 19-20 Oct 2019 | SSCBS, DU | Winner, Best Jugaad Title |
 10| [IncubateIND Innovation Series Hackathon Powered by Microsoft](https://twitter.com/IncubateIND/status/1208667559144218624) | 21-22 December 2019 | Microsoft Gurgaon | 3rd Place Win |
 11| [Innovacer HackerCamp 2019](https://twitter.com/innovaccer/status/1197567144738738176) | 15-16 December 2019 | Innovaccer Noida | Top 15 |
 12| [h@ckGTBIT2.0](https://helloworldofficial.in/hack-gtbit-2.0) | 18-19 January | GTBIT | Best B-Plan Title Winners |
-13| [Build For Digital India by Google, MHRD and MeiTy](https://events.withgoogle.com/buildfordigitalindia/bfdi-finalists/#content) | 21-22 January 2020 | Google Gurgaon | Top 15 pan India; Final Results pending  |
+13| [Build For Digital India by Google, MHRD and MeiTy](https://events.withgoogle.com/buildfordigitalindia/bfdi-finalists/#content) | 21-22 January 2020 | Google Gurgaon | Top 15 pan India  |
```

Comment:
This is okay, thanks

Reply:
That's great. If you need some changes, just mention my name in that issue